Adyen/UI/List/ListItem.swift

```diff
@@ -63,7 +69,9 @@ public class ListItem: FormItem {
         style: ListItemStyle = ListItemStyle(),
         identifier: String? = nil,
         accessibilityLabel: String? = nil,
-        selectionHandler: (() -> Void)? = nil
+        selectionHandler: (() -> Void)? = nil,
+        isSelectable: Bool? = nil,
```

Comment:
Is this property needed? We could decide whether or not it's selectable by checking if a selectionHandler is provided maybe?